e2e/rbd_helper.go

```diff
@@ -1094,6 +1104,31 @@ func getImageInfo(f *framework.Framework, imageName, poolName string) (imageInfo
 	return imgInfo, nil
 }
 
+// getImageStatus queries rbd about the given image and returns its metadata, and returns
+// error if provided image is not found.
+func getImageStatus(f *framework.Framework, imageName, poolName string) (imageStatus, error) {
+	// rbd --format=json status [image-spec | snap-spec]
+	var imgStatus imageStatus
+
+	stdOut, stdErr, err := execCommandInToolBoxPod(
+		f,
+		fmt.Sprintf("rbd status %s %s --format json", rbdOptions(poolName), imageName),
+		rookNamespace)
+	if err != nil {
+		return imgStatus, fmt.Errorf("failed to get rbd status: %w", err)
+	}
+	if stdErr != "" {
+		return imgStatus, fmt.Errorf("failed to get rbd status: %v", stdErr)
+	}
+	err = json.Unmarshal([]byte(stdOut), &imgStatus)
+	if err != nil {
+		return imgStatus, fmt.Errorf("unmarshal failed: %w. raw buffer response: %s",
```

Comment:
don't return the buffer in the error message, as that may get printed many more times when callers log the error. Instead, use `framework.Logf()` to report details like this.

Reply:
done

Reply:
```
e2e/rbd_helper.go:1125:3: k8s.io/kubernetes/test/e2e/framework.Logf does not support error-wrapping directive %w
```
Failed to run container: https://url.corp.redhat.com/cc13cdb

Reply:
`framework.Logf()` does not support `%w` for logging errors. Use `%v` while logging, but when `%w` with `fmt.Errorf()` or `errors.New()`.